app/stac_api/utils.py

```diff
@@ -183,6 +190,25 @@ def harmonize_post_get_for_search(request):
             query_param['bbox'] = json.dumps(query_param['bbox']).strip('[]')  # to string
         if 'query' in query_param:
             query_param['query'] = json.dumps(query_param['query'])  # to string
+
+        accepted_query_parameters = [
+            "bbox", "collections", "datetime", "ids", "intersects", "limit", "query"
+        ]
+        # make sure that all queried parameters (list(query_param.keys())) are in
+        # the accepted_query_parameters
+        if not all(
+            parameter in accepted_query_parameters for parameter in list(query_param.keys())
+        ):
+            wrong_query_parameters = np.setdiff1d(
+                list(query_param.keys()), accepted_query_parameters
+            ).tolist()
+            logger.error(
+                'Query contains the non-allowed parameter(s): %s', str(wrong_query_parameters)
+            )
+            message = f"The query contains the following non-queriable" \
+            f"parameter(s): {str(wrong_query_parameters)}."
+            raise ValidationError(code='query-invalid', detail=_(message))
```

Comment:
For code clarity and avoiding clash between django ValidationError and Rest Framework ValidationError this part of the code should be moved inside a function and put into `validators_serializer.py` we should try to keep all validation code inside the validations modules.

Reply:
`Validators_serializer.py` is where it originally was, before you suggested to move it here ;-) But I can as well move it back again

Reply:
Makes sense to put it in a separate function anyways

Reply:
I originally propose to change the code execution context into harmonize_post_get_for_search
 ;-).

Reply:
I see, makes sense, I'll implement it that way this morning. Function in validations.py, call here. 
😀